### PR TITLE
Merge files tee_mm and tee_mm_unpg

### PR DESCRIPTION
**Pull request by pascal-brand-st-dev**
Signed-off-by: Pascal Brand <pascal.brand@st.com>

### CODE REVIEW

**Comment by jenswi-linaro** [comment]
```
Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>
```

File: core/include/mm/tee_mm.h [hunk 1]
```diff
@@ -30,47 +30,89 @@
...
+extern tee_mm_pool_t tee_mm_vcore;
+
+/*
+ * Returns a pointer to the mm covering the supplied address,
+ * if no mm is found NULL is returned.
+ */
+tee_mm_entry_t *tee_mm_find(const tee_mm_pool_t *pool, uint32_t addr);
+
+/*
+ * Validates that a address (addr) is part of the secure virtual memory
```
Comment (jbech-linaro) [inline core/include/mm/tee_mm.h]: Nitpick "a address" -> "an address".

File: core/include/mm/tee_mm.h [hunk 2]
```diff
@@ -30,47 +30,89 @@
...
+ * Returns false if not valid, true if valid
+ * NOTE: This function is executed in abort mode. Pls take care of stack usage
```
Comment (jbech-linaro) [inline core/include/mm/tee_mm.h]: Pls -> Please

File: core/include/mm/tee_mm.h [hunk 3]
```diff
@@ -30,47 +30,89 @@
...
+ */
+static inline bool tee_mm_validate(const tee_mm_pool_t *pool, uint32_t addr)
+{
+	return tee_mm_find(pool, addr) != 0;
+}
+
+/* Returns virtual address of start of allocated memory for the mm entry. */
```
Comment (jbech-linaro) [inline core/include/mm/tee_mm.h]: Returns __a__ virtial address __to the__ start of __the__ allocated ...

**Comment by jbech-linaro** [comment]
With the minor comments fixed:
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Pull request merged**